src/main/java/mate/academy/model/Movie.java

```diff
@@ -1,10 +1,10 @@
 package mate.academy.model;
 
-import javax.persistence.Entity;
-import javax.persistence.GeneratedValue;
-import javax.persistence.GenerationType;
-import javax.persistence.Id;
-import javax.persistence.Table;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
```

Comment:
I understand what you are doing, but I think it's just better to go with version that is already used in project
latter in tasks you will be given projects with 30+ classes already created. would you change them all?